localization/languages/pt-BR.json

```diff
@@ -31,27 +31,27 @@
         /* custom !bangs
         these are some of the items that show up when you press ! in the searchbar.
         Each one of these strings describes what the command will do when you run it. */
-        "showMoreBangs": null, //missing translation
+        "showMoreBangs": "Mostrar mais...",
         "viewSettings": "Ver configurações",
         "viewReadingList": "Ver listas de leitura",
         "takeScreenshot": "Capturar tela",
-        "clearHistory": "Limpar todo o histórico",
-        "clearHistoryConfirmation": null, //missing translation
+        "clearHistory": "Limpar histórico",
+        "clearHistoryConfirmation": "Limpar todo o histórico e dados de navegação?",
         "switchToTask": "Trocar para a tarefa",
-        "createTask": "Criar uma tarefa",
-        "closeTask": null, //missing translation
+        "createTask": "Criar tarefa",
+        "closeTask": "Fechar tarefa",
         "moveToTask": "Mover esta aba para uma tarefa",
-        "nameTask": null, //missing translation
-        "addBookmark": null, //missing translation
+        "nameTask": " Nomear tarefa",
+        "addBookmark": "Adicionar favorito",
 		"searchBookmarks": "Buscar nos favoritos",
-        "bookmarksAddTag": null, //missing translation
+        "bookmarksAddTag": "Adicionar tag",
 		"bookmarksSimilarItems": null, //missing translation
```

Comment:
what did you mean by "bookmarksSimilarItems" ?

Reply:
When you search for a set of bookmark tags, it's a heading that appears above bookmarks that Min thinks are related to the tags you've entered:

<img src="https://user-images.githubusercontent.com/10314059/69779719-95788600-116e-11ea-997b-63936dd96387.png"/>